### ISSUE TITLE
t/all-basic-operations.t fails with Redis 2.8.19 / CentOS 7

### ISSUE THREAD

**Issue by genio** [jhthorsen/mojo-redis2#30]
This is perl 5, version 16, subversion 3 (v5.16.3) built for x86_64-linux-thread-multi
(with 29 registered patches, see perl -V for more detail)

```
$ yum info redis
Installed Packages
Name        : redis
Arch        : x86_64
Version     : 2.8.19
Release     : 2.el7
Size        : 1.1 M
Repo        : installed
From repo   : Epel-64
```

```
t/all-basic-operations.t .. 1/?
#   Failed test 'Geo did not fail'
#   at t/all-basic-operations.t line 22.
#          got: '[GEOADD redis2:test:Geo -3.055344 53.815931 Blackpool -2.700635 53.759607 Preston -3.006270 53.647920 Southport -3.014240 53.919310 Fleetwood -2.799996 54.045655 Lancaster -0.798977 51.214957 Farnham] ERR unknown command 'GEOADD' at /home/webuser/.cpanm/work/1512067307.16816/Mojo-Redis2-0.29/blib/lib/Mojo/Redis2.pm line 222.
# '
#     expected: ''

#   Failed test 'Hashes did not fail'
#   at t/all-basic-operations.t line 22.
#          got: '[HSTRLEN redis2:test:Hashes bar] ERR unknown command 'HSTRLEN' at /home/webuser/.cpanm/work/1512067307.16816/Mojo-Redis2-0.29/blib/lib/Mojo/Redis2.pm line 222.
# '
#     expected: ''
# Looks like you failed 2 tests of 161.
t/all-basic-operations.t .. Failed 2/161 subtests
	(5 TODO tests unexpectedly succeeded)
```

**Comment by jhthorsen** [comment]
I wonder if this is fixed by #31. Can you try to install `master`?

```
cpanm https://github.com/jhthorsen/mojo-redis2/archive/master.tar.gz
```

**Comment by genio** [comment]
That appears to have the tests passing:
```
$ cpanm --test-only https://github.com/jhthorsen/mojo-redis2/archive/master.tar.gz
--> Working on https://github.com/jhthorsen/mojo-redis2/archive/master.tar.gz
Fetching https://github.com/jhthorsen/mojo-redis2/archive/master.tar.gz ... OK
Configuring mojo-redis2-master ... OK
==> Found dependencies: Sort::Versions
--> Working on Sort::Versions
Fetching http://www.cpan.org/authors/id/N/NE/NEILB/Sort-Versions-1.62.tar.gz ... OK
Configuring Sort-Versions-1.62 ... OK
Building and testing Sort-Versions-1.62 ... OK
Successfully tested Sort-Versions-1.62
Building and testing Mojo-Redis2-0.29 ... OK
Successfully tested Mojo-Redis2-0.29
```

**Comment by Grinnz** [comment]
This should be fixed in 0.30
